cli/src/aws_sso_util/populate_profiles.py

```diff
@@ -149,6 +149,23 @@ def trim_formatter(i, n, **kwargs):
         return formatter(i, n, **kwargs)
     return trim_formatter
 
+def get_field_name_case_formatter(field, transform, formatter):
+    def case_formatter(i, n, **kwargs):
+        LOGGER.debug("evaluating case transform(%s)", transform)
```

Comment:
Let's remove the logger statements in here, they should either be in all the formatters or none, so it would be a separate PR